docs/source/test_set.rst

```diff
@@ -82,4 +86,19 @@ is not available at the time your model was declared.
     trainer.test(test_dataloaders=test)
 
 You can either pass in a single dataloader or a list of them. This optional named
-parameter can be used in conjunction with any of the above use cases.
+parameter can be used in conjunction with any of the above use cases. Additionally,
+you can also pass in an :ref:`datamodules` that have overridden the
+`test_dataloader` method.
```

Comment:
```suggestion
``test_dataloader`` method.
```

Reply:
is there a way to link the docs for test_dataloader from the datamodule docs here? https://pytorch-lightning.readthedocs.io/en/latest/datamodules.html#test-dataloader

Reply:
@rohitgr7 I have added a label to the datamodule docs, which is now linked to